README.md

```diff
@@ -34,12 +33,26 @@ The core approach taken is to strongly encourage and enforce separation of conce
     1. manually run against all files with `pre-commit run --all-files` or against staged files with `pre-commit run`.
     1. Otherwise pre-commit will now run automatically when you make a new commit.
 
+## Checking Enviroment
+Run the following command  (check help string for more details):
+
+```shell
+./vse-sync-collection-tools env verify --interface="<ptp interface>" --kubeconfig="${KUBECONFIG}"
```

Comment:
We should add an example build step if we are giving commands using the binary.

Reply:
Good point